gcp-out-of-cluster.md

```diff
@@ -60,3 +61,24 @@ Container:  "kubernetes_container":  &lt;container>
 To use an alternative or existing label schema for GCP cloud assets, you may supply these in your [values.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml) under the "kubecostProductConfigs.labelMappingConfigs.\<aggregation\>\_external_label" 
 
 > Note: Google generates special labels for GKE resources (e.g. "goog-gke-node", "goog-gke-volume"). Values with these labels are excluded from out-of-cluster costs because Kubecost already includes them as in-cluster assets. Thus, to make sure all cloud assets are included, we recommend installing Kubecost on each cluster where insights into costs are required.
+
+## Cross-Project Service Account Configuration
+
+Due to organizational constraints, it is common that Kubecost must be run in a separate project from the project containing the billing data Big Query dataset which is needed for Cloud Integration. It is still possible to configure Kubecost in this scenario, but some of the values in the above script will need to be changed. First you will need the project id of the projects where Kubecost is installed and where the Big Query dataset are located. Additionally you will need a gcloud user with the permissions `iam.serviceAccounts.setIamPolicy` for the kubecost project and and the ability to manage the roles listed above for the Big Query Project. With these fill in the following script to set the relevant variables:
```

Comment:
Suggestion: "...for the **Kubecost** project and **(remove duplicate and)** the ability..."